src/components/Hero/Hero.mdx

```diff
@@ -381,6 +381,20 @@ the `backgroundImageSrc` will take precedence.
 Use the `textBackgroundColor` prop to set a custom background color for the text
 content.
 
+<Banner
+  content={
+    <>
+      <strong>RECOMMENDATION:</strong> The <code>textBackgroundColor</code> prop
+      and the <code>backgroundColor</code> prop do the same thing for the
+      "tertiary" variant, but it is recommended to use the{" "}
+      <code>textBackgroundColor</code> prop for consistency.
+    </>
+  }
+  mt="l"
+  mb="l"
+  variant="recommendation"
+/>
+
```

Comment:
I am guessing we have to keep `backgroundColor` for backwards compatibility (or is it still being used for certain variants)? Also, since we are now extending props to include style props, it will never be fully deprecated, right?

My question is whether we need to continue "advertising" it at all? Like in the props table, or can we just have one prop – `textBackgroundColor` – we use for this purpose. Then, we probably wouldn't need this note.